charts/ctlog/templates/_helpers.tpl

```diff
@@ -98,10 +98,18 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 Selector labels
 */}}
 {{- define "ctlog.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "ctlog.name" . }}
+app.kubernetes.io/name: {{ .Values.name | quote }}
```

Comment:
im not sure we should change this, as `include "ctlog.name"` already provides a way for you to override it by passing `nameOverride`

https://github.com/sigstore/helm-charts/blob/b1bf44470208d13bcf16830daffe8a9f106a5cd0/charts/ctlog/templates/_helpers.tpl#L5